ruby/lib/trailer_vote/media_types/place.rb

```diff
@@ -29,6 +29,20 @@ class Place < BaseText
           end
         end
 
+        version 3 do
+          attribute :place do
+            attribute :name, String
+            attribute :parent_place, AllowNil(Types::UuidV4)
+            attribute :expires_at, AllowNil(String)
+            attribute :updated_at, Types::Iso8601
+
+            link :self
+            link :products_archive
+            link :parent, allow_nil: true
+            link :children, allow_nil: true
```

Comment:
Children link cannot be nil right? There should always be a resource that shows all the children, it might be empty but it cannot not exist.